hermes-ffi/src/protocol_handler/structures.rs

```diff
@@ -106,6 +106,8 @@ macro_rules! generate_structures_c_symbols {
             $crate::generate_facade_publish!(hermes_injection_publish_injection_status_request = CInjectionFacade: publish_injection_status_request());
             $crate::generate_facade_subscribe!(hermes_injection_subscribe_injection_status = CInjectionFacade: subscribe_injection_status(|CInjectionStatusMessage|));
 
+            $crate::generate_facade_publish!(hermes_tts_publish_register_sound = CTtsFacade: publish_register_sound(CRegisterSoundMessage));
```

Comment:
You forgot the json API ?

Reply:
the api is a binary sound, json is a non sense